include/dfu/dfu_target.rst

```diff
@@ -39,6 +39,11 @@ For bootloader updates, the new firmware is placed in the non-active partition (
 When the complete transfer is done, call the :cpp:func:`dfu_target_done` function to mark the firmware as ready to be booted.
 On the next reboot, the device will run the new firmware.
 
+.. note::
```

Comment:
```
.. note::
   To maintain the write progress in case the device reboots, enable the configuration options :option:`CONFIG_SETTINGS` and :option:`CONFIG_DFU_TARGET_MCUBOOT_SAVE_PROGRESS`.
   The MCUboot target then uses the settings subsystem in Zephyr to store the current progress of the :cpp:func:`dfu_target_write` function across power failures and device resets.
```

Reply:
Can we add a link to the settings subsystem? Or in other words, is there some documentation on that subsystem in the Zephyr documentation?

Reply:
It is here https://docs.zephyrproject.org/latest/reference/runtime_conf/settings/settings.html can we link directly with some `:ref` or does it need full link since it is in zephyr?

Reply:
:ref:`zephyr:settings` should work. :)

Reply:
tried to add `:ref:zephyr:settings` hope that works

Reply:
Can you use the changes from the first comment as well? ;)

Reply:
didn't see it will do

Reply:
Added